models/api.models.js

```diff
@@ -47,9 +47,23 @@ exports.fetchCommentsByArticleId = (id) => {
 	}
 	let sqlQuery = `SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC`;
 	return db.query(sqlQuery, [id]).then((result) => {
-		if (result.rows.length === 0) {
+		return result.rows;
+	});
+};
+
+exports.postCommentByArticleId = (username, body, id) => {
+	if (isNaN(id)) {
```

Comment:
If the artilce_id is not a number you will get a psql error thrown so you can potentially handle this as a psql error rather than custom